src/libxrpl/protocol/TxMeta.cpp

```diff
@@ -213,6 +217,8 @@ TxMeta::getAsObject() const
 {
     STObject metaData(sfTransactionMetaData);
     XRPL_ASSERT(mResult != 255, "ripple::TxMeta::getAsObject : result is set");
+    if (mBatchId)
+        metaData.setFieldH256(sfBatchTransactionID, mBatchId.value());
```

Comment:
Thoughts on whether this should be called something like `ParentBatchID` instead?

Reply:
Thats fine